### PR TITLE
One fix per day #3

### PR DESCRIPTION
**Pull request by mbaechler**

### CODE REVIEW

**Comment by chibenwa** [comment]
test this please

**Comment by Arsnael** [comment]
```
[bda6a3c505d564bede1aafe5ae2d1a1ce06e0dcc] [ERROR] src/main/java/org/apache/james/imap/processor/ListProcessor.java:[30] (imports) CustomImportOrder: Wrong lexicographical order for 'org.apache.james.imap.api.display.HumanReadableText' import. Should be before 'org.apache.james.imap.api.display.ModifiedUtf7'.
[bda6a3c505d564bede1aafe5ae2d1a1ce06e0dcc] [ERROR] src/main/java/org/apache/james/imap/processor/LSubProcessor.java:[29] (imports) CustomImportOrder: Wrong lexicographical order for 'org.apache.james.imap.api.display.HumanReadableText' import. Should be before 'org.apache.james.imap.api.display.ModifiedUtf7'.
[bda6a3c505d564bede1aafe5ae2d1a1ce06e0dcc] [ERROR] src/main/java/org/apache/james/imap/decode/ImapRequestLineReader.java:[39] (imports) CustomImportOrder: Wrong lexicographical order for 'org.apache.james.imap.api.display.HumanReadableText' import. Should be before 'org.apache.james.imap.api.display.ModifiedUtf7'.
```

**Comment by chibenwa** [comment]
```
[fd3ab9d9c32023e88d7bba93aadfd8d9abc7ba38] [ERROR]   CassandraMailRepositoryTest.storingAndRemovingMessagesConcurrentlyShouldLeadToConsistentResult » Execution
[fd3ab9d9c32023e88d7bba93aadfd8d9abc7ba38] [ERROR]   CassandraMailRepositoryTest.storingAndRemovingMessagesConcurrentlyShouldLeadToConsistentResult » Execution
[fd3ab9d9c32023e88d7bba93aadfd8d9abc7ba38] [ERROR]   CassandraMailRepositoryTest.storingAndRemovingMessagesConcurrentlyShouldLeadToConsistentResult » Execution
[fd3ab9d9c32023e88d7bba93aadfd8d9abc7ba38] [INFO] 
```

test this please

File: server/data/data-library/src/main/java/org/apache/james/mailrepository/lib/AbstractMailRepository.java [hunk 1]
```diff
@@ -40,8 +40,7 @@
 import com.google.common.collect.Iterators;
 
 /**
- * This class represent an AbstractMailRepository. All MailRepositories should
- * extend this class.
+ * This class represent an AbstractMailRepository that may help implementing MailRepository interface
```
Comment (rouazana) [inline server/data/data-library/src/main/java/org/apache/james/mailrepository/lib/AbstractMailRepository.java]: represents

Comment (mbaechler) [reply server/data/data-library/src/main/java/org/apache/james/mailrepository/lib/AbstractMailRepository.java]: fixed

**Comment by mbaechler** [comment]
merged

**Pull request closed without merging**